plugins/org.yakindu.sct.doc.user/src/user-guide/overview.textile

```diff
@@ -3,81 +3,86 @@ p.
 
 h1(#overview_what_are_state_machines). What is a state machine?
 
-A state machine is a behavior model. It consists of different so called states. Because the amount of states of a state machine is finite it is called finite state machine (FSM). There are basic types like Mealy and Moore machines and more complex types like Harel and UML statecharts. 
+A state machine is a behavior model. It consists of different so called states. Because the amount of states of a state machine is finite it is called finite state machine (FSM). There are basic types like Mealy and Moore machines and more complex types like Harel and UML statecharts.
 
 This introduction gives a short overview of the common basis and the differences between state machine types.
 
-!(large)images/simple_moore.png(Simple example as a Moore machine)!
+!(small)images/simple_moore.png(Simple example as a Moore machine)!
 
 p=. Simple state machine
 
-In automata theory a state is a situation of a system depending on previous inputs and causes a reaction on following inputs. Automata theory uses the terms input/output for simple text input/output. Modern state machines use an extended definition of inputs and outputs. These can be events or more complex reactions than simply printing a certain text. Another important element of  state machines are transitions. A transition, as the name says, describes the transition from one state to another.
+The basic building blocks of a state machine are states and transitions. A state is a situation of a system depending on previous inputs and causes a reaction on following inputs. Transitions define for which external input and under which condition a state change occurs.
```

Comment:
I stay at my point: I would not change the old introduction.

This is simple and on point: "A transition, as the name says, describes the transition from one state to another."
For example, this is how Wikipedia describes it: "the change from one state to another is called a **transition"

This leaves questions: "transitions define for which external input and under which condition a state change occurs."
- What's the difference between external and internal? Why external?
- What is a state change?